plugins/hybrid_ae_pkg/hybrid_ae_templates/hybrid_ae_template.html

```diff
@@ -0,0 +1,51 @@
+{% import 'forms.html' as forms %}
+
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+
+    <style type="text/css">
+        @import url("{{url_for('static', filename='forms.css')}}");
+    </style>
+</head>
+
+<body>
+    <div class="qhana-microfrontend">
+
+        <details class="qhana-help">
```

Comment:
Some of this help text can be moved into the input model metadata (the descriptions are now displayed below the form fields and support [limited] markdown).

Reply:
Ok, I fixed it. Probably not necessary to have the example values included at all as there is the option to fill out example values.